src/main/java/com/angelbroking/smartapi/orderupdate/OrderUpdateWebsocket.java

```diff
@@ -0,0 +1,163 @@
+package com.angelbroking.smartapi.orderupdate;
+
+import com.angelbroking.smartapi.Routes;
+import com.angelbroking.smartapi.smartstream.models.SmartStreamError;
+import com.angelbroking.smartapi.utils.Utils;
+import com.neovisionaries.ws.client.*;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+import java.util.Timer;
+import java.util.TimerTask;
+
+@Slf4j
+public class OrderUpdateWebsocket {
+    private static final int PING_INTERVAL = 10000; // 10 seconds
+    private static final String HEADER = "Authorization";
```

Comment:
header name should be HEADER_AUTHORIZATION

Reply:
Header variable is changed accordingly. Please check